lib/hashie/extensions/dash/indifferent_access.rb

```diff
@@ -11,6 +11,7 @@ module ClassMethods
           # Check to see if the specified property has already been
           # defined.
           def property?(name)
+            name = defined?(translations) && translation_exists?(name) ? translations[:"#{name}"] : name
```

Comment:
Can `:"#{name}"` be written as name.to_sym? Avoiding doing it twice for a symbol.